release-notes/release-notes59.md

```diff
@@ -28,7 +28,11 @@ keywords: releasenotes bugfixes
 
 The following new features were added for the {{site.productname}} 5.9 release.
 
-### Feature name
+### New `toolbar_sticky_offset` option for customizing toolbar sticky
+
+The new `toolbar_sticky_offset` setting was added to allow the toolbar to stick at a specific offset from the top/bottom of the view (depending on the toolbar mode).
+
+For information on the `toolbar_sticky` option, see: [User interface options - toolbar_sticky]({{site.baseurl}}/configure/editor-appearance/#toolbar_sticky).
```

Comment:
This should link to the documentation for the new setting instead, which needs to be added as part of this PR. 

So we need an include added here (directly after the toolbar_sticky include): [editor-apperance.md](https://github.com/tinymce/tinymce-docs/blob/main/configure/editor-appearance.md) and then a new setting file added here: [configuration](https://github.com/tinymce/tinymce-docs/tree/main/_includes/configuration) (see [toolbar_sticky.md](https://github.com/tinymce/tinymce-docs/blob/main/_includes/configuration/toolbar_sticky.md) for an example). We'll also need the nav entry added here: [nav.yml](https://github.com/tinymce/tinymce-docs/blob/main/_data/nav.yml#L128)